src/utilities/geometry/FloorplanJS.cpp

```diff
@@ -1078,11 +1078,10 @@ ThreeScene FloorplanJS::toThreeScene(bool openstudioFormat) const {
   for (Json::ArrayIndex storyIdx = 0; storyIdx < storyN; ++storyIdx) {
 
     if (checkKeyAndType(stories[storyIdx], "name", Json::stringValue)) {
-      std::string storyName = stories[storyIdx].get("name", storyName).asString();
-      buildingStoryNames.push_back(storyName);
+      buildingStoryNames.push_back(stories[storyIdx].get("name", "").asString());
```

Comment:
The old notation was definitely extremely weird to look at. It's weird that this compiles, but I guess we shush some warnings, and especially we use `-Wno-uninitialized`